servo-build-dependencies/ci.sls

```diff
@@ -0,0 +1,14 @@
+{% from 'common/map.jinja' import root %}
+{% from tpldir ~ '/map.jinja' import sccache %}
+
+sccache:
+  file.managed:
+    - name: {{ sccache.destination }}
+    - source: https://s3.amazonaws.com/rust-lang-ci/rust-ci-mirror/{{ sccache.version }}-sccache-{{ sccache.platform }}
```

Comment:
Official binary releases are [not yet available](https://github.com/servo/servo/issues/17106#issuecomment-306148322) for sccache, so per suggestion I copied these links from [rust-lang/rust](https://github.com/rust-lang/rust) CI scripts.